src/bp-performance/classes/integrations/class-bb-media-albums.php

```diff
@@ -0,0 +1,385 @@
+
+			// Added moderation support.
+			'bp_suspend_media_suspended',         // Any Media Suspended.
+			'bp_suspend_media_unsuspended',       // Any Media Unsuspended.
+			'bp_suspend_media_album_suspended',   // Any Media Album Suspended.
+			'bp_suspend_media_album_unsuspended', // Any Media Album Unsuspended.
+		);
+
+		/**
+		 * Add Custom events to purge media albums endpoint cache
+		 */
+		$purge_events = apply_filters( 'bbplatform_cache_bp_media_albums', $purge_events );
+		$this->purge_event( 'bp-media-albums', $purge_events );
+
+		/**
+		 * Support for single items purge
+		 */
+		$purge_single_events = array(
+			'bp_media_album_after_save'          => 1, // Any Media Album add.
+			'bp_media_album_after_delete'        => 1, // Any Media Album deleted.
+
+			'bp_media_add'                       => 1, // Any Media Photo add.
+			'bp_media_after_save'                => 1, // Any Media Photo updated.
+			'bp_media_before_delete'             => 1, // Any Media Photos deleted.
+
+			// Media group information update support.
+			'groups_update_group'                => 1,   // When Group Details updated.
+			'groups_group_after_save'            => 1,   // When Group Details save.
+			'groups_group_details_edited'        => 1,   // When Group Details updated form Manage.
+
+			// Added moderation support.
+			'bp_suspend_media_suspended'         => 1, // Any Media Suspended.
+			'bp_suspend_media_unsuspended'       => 1, // Any Media Unsuspended.
+			'bp_suspend_media_album_suspended'   => 1, // Any Media Album Suspended.
+			'bp_suspend_media_album_unsuspended' => 1, // Any Media Album Unsuspended.
+
+			// Add Author Embed Support.
+			'profile_update'                     => 1, // User updated on site.
+			'deleted_user'                       => 1, // User deleted on site.
+			'xprofile_avatar_uploaded'           => 1, // User avatar photo updated.
+			'bp_core_delete_existing_avatar'     => 1, // User avatar photo deleted.
+		);
+
+		/**
+		 * Add Custom events to purge single album endpoint cache
+		 */
+		$purge_single_events = apply_filters( 'bbplatform_cache_bp_media_albums_single', $purge_single_events );
+		$this->purge_single_events( 'bbplatform_cache_purge_bp-media_albums_single', $purge_single_events );
+
+		$is_component_active = Helper::instance()->get_app_settings( 'cache_component', 'buddyboss-app' );
+		$settings            = Helper::instance()->get_app_settings( 'cache_bb_media', 'buddyboss-app' );
+		$cache_bb_media      = isset( $is_component_active ) && isset( $settings ) ? ( $is_component_active && $settings ) : false;
+
+		if ( $cache_bb_media ) {
+
+			$this->cache_endpoint(
+				'buddyboss/v1/media/albums',
+				Cache::instance()->month_in_seconds * 60,
+				$purge_events,
+				$event_groups,
+				array(
+					'unique_id'         => 'id',
+					'purge_deep_events' => array_keys( $purge_single_events ),
+				),
+				true
+			);
+
+			$this->cache_endpoint(
+				'buddyboss/v1/media/albums<id>',
```

Comment:
@KartikSuthar please check this endpoint path. slash should be there before`<id>`  like this `'buddyboss/v1/media/albums/<id>'`

Reply:
Done